events/message.js

```diff
@@ -22,7 +23,8 @@ module.exports = async (client, message) => {
 
 	// Check the message for profanity
 	profanityActions.checkForProfanity(client, message);
-
+	// Handle greetings
+	greetinAction.reactToGreeting(client, message);
```

Comment:
Typo: Should be greeting instead of greetin

Reply:
Fixed